odoo/models.py

```diff
@@ -830,14 +830,15 @@ def load(self, fields, data):
         ModelData.clear_caches()
         extracted = self._extract_records(fields, data, log=messages.append)
         converted = self._convert_records(extracted, log=messages.append)
+        unknown_msg = _(u"Unknown database error: '%s'")
         for id, xid, record, info in converted:
             try:
                 cr.execute('SAVEPOINT model_load_save')
             except psycopg2.InternalError as e:
                 # broken transaction, exit and hope the source error was
                 # already logged
                 if not any(message['type'] == 'error' for message in messages):
-                    messages.append(dict(info, type='error',message=u"Unknown database error: '%s'" % e))
+                    messages.append(dict(info, type='error', message=unknown_msg % e))
```

Comment:
@mart-e previously this message wasn't translated, means we need to add it to the pot right?

Reply:
as we are in the context of a psycopg2 error, we probably don't have a valid cursor, it is fine to keep it untranslated

Reply:
That's why I cheated by putting the msg above the cursor fuckery. 
So I tested every case and with these changes, every single message was translated fully.

If you want to have a check, there are a lot of commits that forgot to update the pot, so there's a separate commit for that.

Reply:
oh right, I didn't read properly the issue. So yes, thanks for the update of the .pot